### ISSUE TITLE
fix 32-bit ppc support in Firefox

### ISSUE THREAD

**Issue by q66** [void-ppc/void-packages#28]
There are several issues here:

1) `nodejs` - https://github.com/void-ppc/void-packages/issues/27 For now, this can be worked around by switching to `nodejs-lts`. (for cross-compiling it's a non-issue as node is a host dependency)
2) ~~There's some C++ code paths that fail to compile.~~
3) ~~`rustc` crashes when compiling `style` during native compilation; cross compiling however works.~~ this has been fixed some time ago
4) ~~WebRTC failed to build (could be disabled in worst case)~~ worked around by disabling
5) Crashes on startup in `xptcall`

**Comment by q66** [comment]
I wrote fixes for the C++ failures: https://github.com/void-ppc/void-packages/commit/a6e4022974d41de4d454b2cba7b7b15771ecc3fd which also disable WebRTC on ppc32 as it's not easily fixable at this time.

It is now possible to cross-compile Firefox for 32-bit ppc from another environment (native compilation does not work as `rustc` crashes). However, it still segfaults upon launch, I managed to capture a backtrace: https://gist.github.com/q66/0aefa2d42a846cfceb84607f19018c86

I'm not quite sure what to do with this one.
